### ISSUE TITLE
Database Testing

### ISSUE THREAD

**Issue by ahberndt** [Angel0117/Application_in_Information_Systems_Project#6]
When uploading in bulk meteor is throwing an error stating " MongoError: Invalid Operation, no operations specified" 

One thing I would also note is that you should be bulk uploading to collection info2 not (from what I can tell) "bulktest" 

I was also running into issues getting your custom querying to work properly.  It wasn't querying properly it seemed like it was querying from info1 not info2.

**Comment by CMelhaire** [comment]
Yes - we had (have) some difficulty getting our bulk upload to function.  We had it set to bulktest for debugging, but we can definitely switch that back out.  I think we'll finally be able to get bulk upload working by this evening.

I'll check into the custom querying as well, that's kind of weird.  I'll see if we did something strange with our code.
